programs/timing.py

```diff
@@ -70,3 +70,18 @@ def _monotonic_time_nanos_cygwin():
         k32.QueryPerformanceCounter(ctypes.byref(perf_counter))
         return perf_counter.value * NSEC_PER_SEC / perf_frequency.value
     monotonic_time_nanos = _monotonic_time_nanos_cygwin
+elif platform.system() == 'FreeBSD':
+    CLOCK_MONOTONIC = 4
+    # On FreeBSD9 and FreeBSD10
+    libc = ctypes.CDLL('libc.so.7', use_errno=True)
```

Comment:
> On 16 May 2016, at 13:02, Coneko <notifications@github.com> wrote:
> 
> > @@ -70,3 +70,18 @@ def _monotonic_time_nanos_cygwin():
> >          k32.QueryPerformanceCounter(ctypes.byref(perf_counter))
> >          return perf_counter.value * NSEC_PER_SEC / perf_frequency.value
> >      monotonic_time_nanos = _monotonic_time_nanos_cygwin
> > +elif platform.system() == 'FreeBSD':
> > +    CLOCK_MONOTONIC = 4
> > +    # On FreeBSD9 and FreeBSD10
> > +    libc = ctypes.CDLL('libc.so.7', use_errno=True)
> 
> Since this is the only difference between this case and the linux case, can you extract the common parts?

One is CLOCK_MONOTONIC, the other is CLOCK_MONOTONIC_RAW (they both happen to have the same integer value). I can update if you want to? Should I rename method names from _linux to _posix?

holger